repos/system_upgrade/el7toel8/actors/scheduleselinuxrelabeling/actor.py

```diff
@@ -23,10 +24,10 @@ def process(self):
                 try:
                     with open('/.autorelabel', 'w'):
                         pass
+                    report_generic(
+                        severity='low',
+                        title='SElinux scheduled for relabelling',
+                        summary='/.autorelabel file touched on root in order to schedule SElinux relabelling',
+                    )
                 except OSError as e:
-                    self.produce(FinalReport(
```

Comment:
why are we no longer reporting error here and only logging it?

Reply:
Yeah this should be still reported, in case something went wrong.

Reply:
@jmikovic  because originally there was not actor that is consuming that message and it was idea to do something like that, when we have reporting.

Reply:
@pirat89 Considering the way how it is now going to be implemented, it's even more encouraging now to NOT remove it, since all reports now can be collected.

Reply:
Does adding inhibitor too make sense? As we found out we cannot log in to the system if it is not relabeled? Same for the other message...

Reply:
@Rezney well it is not an inhibitor anymore by the definition we are using it for. Because this won’t block anything.
Now it’s a high impact problem, but what if there is another problem and this not even is executed, then that would be a case where there wouldn’t be that “inhibitor”

Tl;Dr no idea, @pirat89 what do you think?

Reply:
@vinzenz @Rezney tl;dr it should be high impact without inhibitor flag. Additionally, it is not inhibitor when we are switching into the Permissive mode - so user can do relabeling manually. You should report what user should do (required action after upgrade - as we discussed, these flags are missing yet, but can be added later; just put here something like FIXME so we will find that additional flags should be added).

Inhibitor's should be used "only" before reporting phase (discussable but as I understand inhibitor, that is message consumed by reporting actor before upgrade, it doesn't make sense to use inhibitors from that point after the reporting phase. What do you want to inhibit when even upgrade transaction has been performed already? All checks should be performed during the ChecksPhase - and that's place where you should use inhibitors. After that, we are talking much more about errors as you usually expect that things will work, you are not doing additional checks whether you can do upgrade - excluding networking that needs to be checked additionally from the initramdisk with new kernel).